src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings

```diff
@@ -818,7 +818,7 @@
 
 "ExposureSubmissionWarnOthers_consent_bullet3" = "Вашата самоличност няма да бъде разкрита. Другите потребители няма да узнаят кой е споделил резултата от теста си.";
 
-"ExposureSubmissionWarnOthers_consent_bullet4" = "За да дадете съгласието си, трябва да имате навършени 16 години.";
+"ExposureSubmissionWarnOthers_consent_bullet5" = "За да дадете съгласието си, трябва да имате навършени 16 години.";
```

Comment:
you don't need to add this, as this will be an automatic process, you could simply remove them.

Reply:
This is an old bullet point which was already translated, i changed the number to 5, and added the new bullet into 4

Reply:
should i still remove them ?